DESCRIPTION

```diff
@@ -25,12 +25,11 @@ LazyData: true
 Imports:
```

Comment:
The automated build is failing because you have added a dependency on the boot package but have not added it here as an import. Hope this makes sense.